packages/cysync-core-constants/src/i18n/lang/en.json

```
@@ -293,8 +293,8 @@
         "error": "Insufficient funds",
         "zeroAmount": "Amount can't be zero",
         "notOverDustThreshold": "Amount is lower than dust limit",
-        "amountBelowXrpReserve": "Recipient address is inactive. Send atleast 10 XRP to activate it.",
-        "balanceBelowXrpReserve": "Balance cannot go below 10 XRP"
+        "amountBelowReserveBalance": "Recipient address is inactive. To activate it, send atleast ",
+        "balanceBelowReserveBalance": "Balance cannot go below "
```

```suggestion
        "amountBelowReserveBalance": "Recipient address is inactive. To activate it, send atleast ${amount} ${unit}",
        "balanceBelowReserveBalance": "Balance cannot go below ${amount} ${unit}"
```

Prefer proper format strings, this will result in better translations and future modifications.